frontend/src/views/DiagnosticEditor/index.vue

```diff
@@ -362,14 +333,14 @@ export default {
       }
     },
     originalDiagnostic() {
-      if (this.isNewDiagnostic) return {}
-      return this.canteen.diagnostics.find((diagnostic) => diagnostic.year === parseInt(this.year))
+      if (this.isNewDiagnostic || !this.originalCanteen) return {}
+      return this.originalCanteen.diagnostics.find((diagnostic) => diagnostic.year === parseInt(this.year))
     },
     diagnosticIsUnique() {
       if (!this.isNewDiagnostic) return true
       if (!this.canteenId || !this.diagnostic.year) return true
```

Comment:
Je pense qu'on peut enlever `!this.canteenId` ici, car on va toujours l'avoir

Reply:
Fix ici : https://github.com/betagouv/ma-cantine/pull/793/commits/459b9e41cd25eb790320e527255b79d61ae50c4b